Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/ConceptSearch.java

```diff
@@ -157,4 +226,11 @@ private void createEntryMap(ConceptEntry[] searchResults, Map<ConceptEntry, Conc
             entryMap.put(entry, type);
         }
     }
+
+    private String removeTrailingBackSlash(String val) {
```

Comment:
This method is redundant in ConceptSearch.java and ConceptEntry.java. Please let me know if I need to create a Utility class and add this method.